runtime/include/chpl-file-utils.h

```diff
@@ -36,6 +36,8 @@ qioerr chpl_fs_chown(const char* name, int uid, int gid);
 
 qioerr chpl_fs_copy_metadata(const char* source, const char* dest);
 
+char* chpl_fs_curdir(void);
```

Comment:
Meaning that the declared return type here and in the following function should be "c_string".